packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1263,7 +1263,8 @@ class _SliverAppBarDelegate extends SliverPersistentHeaderDelegate {
 
     final bool isScrolledUnder = overlapsContent || forceElevated || (pinned && shrinkOffset > maxExtent - minExtent);
     final bool isPinnedWithOpacityFade = pinned && floating && bottom != null && extraToolbarHeight == 0.0;
-    final double toolbarOpacity = !pinned || isPinnedWithOpacityFade
+    final bool isAccessibilityOn = MediaQuery.of(context).accessibleNavigation;
```

Comment:
Should the value of MediaQuery.of(context).accessibleNavigation be a passed parameter so it can be checked in shouldRebuild for when it changes?

Reply:
Use of MediaQuery.of or MediaQuery.accessibleNavigationOf  will cause it to rebuild, is it still necessary to add a parameter to  `shouldRebuild`?

https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/media_query.dart#L1457-L1462

Reply:
Can you add a test that switches accessibleNavigation off and on to verify it updates? Usually I would use didChangeDependencies, but _SliverAppBarDelegate is not a stateful widget.

Reply:
Test added. Looks like I do need to update it to shouldRebuild(), thank you!